problemsets-mat223/modules/module7.tex

```diff
@@ -133,7 +168,30 @@
 	\]
 	Find a normal vector for $\mathcal Q$ and write $\mathcal Q$ in normal form.
 
-	XXX Finish
+	Similarly to the above example, since normal vectors for $\mathcal Q$ need to be orthogonal to $\vec d_1=\mat{1\\1\\-1\\1}$, $\vec d_2=\mat{0\\1\\0\\1}$ and $\vec d_3=\mat{2\\0\\0\\0}$, we could find the normal vectors by solving
+	\[
+	\underbrace{\mat{1&1&-1&1\\0&1&0&1\\2&0&0&0}}_A\mat{x\\y\\z\\w}=\mat{0\\0\\0}.
+    \]
+    By row reduction on matrix A as follows:
```

Comment:
At this point we don't need to outline the row reduction steps. You can just say, `Row reducing $A$, we get \[\Rref(A) = ...\]`